examples/client/Locomotion/src/Components/PriceCard/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { View } from 'react-native';
+import { SubText } from '../../pages/Profile/ScreenText/styles';
+
+type PriceCardProps = {
+    name: string,
+    text: string
+}
+
+const PriceCard = ({ name, text } : PriceCardProps) => (
+  <View style={{
+    flexDirection: 'row', marginVertical: 5, marginLeft: 15, width: '100%', justifyContent: 'space-between',
+  }}
+  >
+    <SubText>{name}</SubText>
+    <SubText style={{ textAlign: 'right' }}>
```

Comment:
use styled components and create a new instance for this specific card, don't reuse the one from ScreenText